modules/swagger-codegen/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -44,6 +44,10 @@
             <artifactId>spring-boot-starter-tomcat</artifactId>
             <scope>provided</scope>
         </dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
```

Comment:
@cbornet  I think it is easier to change application.properties than change pom.xml.
If you need persistence (what project do not need) it will change pom for recommender dependency and application.properties for add datasource.